extensions/cornerstone/src/getToolbarModule.tsx

```diff
@@ -176,14 +220,7 @@ export default function getToolbarModule({ commandsManager, servicesManager }: w
       name: 'evaluate.not3D',
       evaluate: ({ viewportId, disabledText }) => {
         const viewport = cornerstoneViewportService.getCornerstoneViewport(viewportId);
-
-        if (viewport?.type === 'volume3d') {
-          return {
-            disabled: true,
-            className: '!text-common-bright ohif-disabled',
-            disabledText: disabledText ?? 'Not available on the current viewport',
-          };
-        }
+        return viewport?.type === 'volume3d' ? getDisabledState(disabledText) : undefined;
```

Comment:
This one can be done just like the evaluate viewport not supported or viewport supported.